js/controller.js

```diff
@@ -0,0 +1,123 @@
+goog.module('wordsearch.controller');
+
+const model = goog.require('wordsearch.model');
+const view = goog.require('wordsearch.view');
+
+/**
+ * The controller bridges the gap between the model and view by handling input
+ * passed from the view and make the appropriate calls to update the model. It
+ * also makes call to updae the view when the model changes.
```

Comment:
spelling - s/updae/update

Reply:
One would think after spelling update so many times in this PR I wouldn't make that mistake. - Updated.